src/network/networkctl.c

```diff
@@ -2305,6 +2368,8 @@ static int help(void) {
                "  lldp [PATTERN...]      Show LLDP neighbors\n"
                "  label                  Show current address label entries in the kernel\n"
                "  delete DEVICES...      Delete virtual netdevs\n"
+               "  up DEVICES...          Up virtual netdevs\n"
+               "  down DEVICES...        Down virtual netdevs\n"
```

Comment:
Is this only for virtual netdevs? I would assume it'd work on all kinds.
"Up" and "down" are not verbs — maybe "Set device state to UP", "Set device state to DOWN", or "Bring devices up", "Bring devices down".

Reply:
oops yes they are for all. Thanks.